REQUIRE

```diff
@@ -1,4 +1,6 @@
 julia 0.4
+Compat 0.8.5
 Distances 0.3.1
+FixedSizeArrays
 StatsBase 0.9.0
-Compat 0.8.5
+PointClouds
```

Comment:
So the dependency on PointClouds is new? I'm not familiar with it. Is it actively maintained? If it's only being used for one function (which also has another method) then it's probably not worth having another dependency.

Also should specify minimum versions for all package requirements.